SPECS/openvswitch.spec

```diff
@@ -33,33 +39,29 @@ Patch14: 0001-bond-Remove-executable-bit-from-bond.c.patch
 Patch15: 0001-lacp-Avoid-packet-drop-on-LACP-bond-after-link-up.patch
 Patch16: 0001-lacp-report-desync-in-ovs-threads-enabling-slave.patch
 Patch17: 0001-ofproto-bond-Improve-admissibility-debug-readability.patch
-Patch18: CA-72973-hack-to-strip-temp-dirs-from-paths.patch
-Patch19: CP-15129-Convert-to-use-systemd-services.patch
-Patch20: CA-78639-dont-call-interface-reconfigure-anymore.patch
-Patch21: CA-141390-dont-read-proc-cpuinfo-if-running-on-xenserver.patch
-Patch22: CA-153718-md5-verification-dvsc.patch
-Patch23: CP-9895-Add-originator-to-login_with_password-xapi-call.patch
-Patch24: CP-13181-add-dropping-of-fip-and-lldp.patch
-Patch25: use-old-db-port-6632-for-dvsc.patch
-Patch26: CA-243975-Fix-openvswitch-service-startup-failure.patch
-Patch27: CP-23098-Add-IPv6-multicast-snooping-toggle.patch
-Patch28: CA-265107-When-enable-igmp-snooping-cannot-receive-ipv6-multicast-traffic.patch
-Patch29: 0001-xenserver-fix-Python-errors-in-Citrix-changes.patch
-Patch30: 0002-O3eng-applied-patch-on-top-of-the-NSX-OVS.patch
-Patch31: 0003-update-bridge-fail-mode-settings-when-bridge-comes-up.patch
-Patch32: CP-23607-inject-multicast-query-msg-on-bond-port.patch
-Patch33: mlockall-onfault.patch
-Patch34: hide-logrotate-script-error.patch
+Patch18: 0001-ofproto-dpif-xlate-Always-mask-ip-proto-field.patch
+Patch19: CA-72973-hack-to-strip-temp-dirs-from-paths.patch
+Patch20: CP-15129-Convert-to-use-systemd-services.patch
+Patch21: CA-78639-dont-call-interface-reconfigure-anymore.patch
+Patch22: CA-141390-dont-read-proc-cpuinfo-if-running-on-xenserver.patch
+Patch23: CA-153718-md5-verification-dvsc.patch
+Patch24: CP-9895-Add-originator-to-login_with_password-xapi-call.patch
+Patch25: CP-13181-add-dropping-of-fip-and-lldp.patch
+Patch26: use-old-db-port-6632-for-dvsc.patch
+Patch27: CA-243975-Fix-openvswitch-service-startup-failure.patch
+Patch28: CP-23098-Add-IPv6-multicast-snooping-toggle.patch
+Patch29: CA-265107-When-enable-igmp-snooping-cannot-receive-ipv6-multicast-traffic.patch
+Patch30: 0001-xenserver-fix-Python-errors-in-Citrix-changes.patch
+Patch31: 0002-O3eng-applied-patch-on-top-of-the-NSX-OVS.patch
+Patch32: 0003-update-bridge-fail-mode-settings-when-bridge-comes-up.patch
+Patch33: CP-23607-inject-multicast-query-msg-on-bond-port.patch
+Patch34: mlockall-onfault.patch
+Patch35: hide-logrotate-script-error.patch
 
 # XCP-ng patches
-# Backported from XS 8
-Patch1000: openvswitch-2.5.3-CVE-2023-1668-ofproto-dpif-xlate-Always-mask-ip-proto-field.XS.patch
-Patch1001: openvswitch-2.5.3-comment-failing-tests.XCP-ng.patch
+Patch1000: openvswitch-2.5.3-comment-failing-tests.XCP-ng.patch
```

Comment:
Is it still required, and why?

Reply:
Yes, still required, because we enable the tests whereas XS disabled them. We should try to contribute this upstream again.

Reply:
Yes it is still needed. There are tests that do not pass, XenServer does not have this as the tests are fully disabled, we added this on our side and went with disabling some tests, so we at least have a part of the coverage, we never took the time to investigate why these tests were failing.